src/backend/marsha/core/api.py

```diff
@@ -406,25 +407,46 @@ def post(self, request):
 
         consumer_site = video.playlist.consumer_site
 
+        # We want to allow to put xapi statement in a logger. We have a here a convention name
+        # specific allowing a have a logger by consumer site. If you want to log xapi statements
+        # you have to prefix the consumer site domain with `xapi.`. For example for the consumer
+        # site with domaine name foo.education you have to create a logger name
+        # `xapi.foo.education` and configure as you want the associated handler and formatter
+        # depending where you want to log the statements.
```

Comment:
Maybe this could be simplified as follow:

```suggestion
        # xapi statements are sent to a consumer-site-specific logger. We assume that the logger
        # name respects the following convention: "xapi.[consumer site domain]", 
        # _e.g._ `xapi.foo.education` for the `foo.education` consumer site domain. Note that this 
        # logger should be defined in your settings with an appropriate handler and formatter.
```